Flashcards.Dejmenek/Flashcards.Dejmenek/DataAccess/DataContext.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Dapper;
+using System.Configuration;
+using System.Data.SqlClient;
+
+namespace Flashcards.Dejmenek.DataAccess
+{
+    public static class DataContext
+    {
+        private static readonly string _connectionString = ConfigurationManager.ConnectionStrings["LocalDbConnection"].ConnectionString;
+
+        public static void CreateDatabase()
+        {
```

Comment:
As a little fun challenge, see how you can create the database when the user runs the application rather than manually? `Note: this is not absolutely necessary but just a challenge. Not doing this will not affect your project's approval. If it is too difficult you can go ahead and skip it 😃 `